t/admin/standalone.spec.ts

```diff
@@ -641,5 +721,228 @@ describe('Admin - Standalone', () => {
           'invalid routes at index 0, err: invalid configuration: failed to match pattern "^((uri|server_name|server_addr|request_uri|remote_port|remote_addr|query_string|host|hostname|mqtt_client_id)|arg_[0-9a-zA-z_-]+)$" with "args_invalid"',
       });
     });
+    it('validate config (duplicate route id)', async () => {
```

Comment:
What is the purpose of these newly added tests? Did we previously have duplicate ID checking functionality without test coverage?